src/dyad/modules/dyad.c

```diff
@@ -41,469 +41,490 @@
 #include <stdlib.h>
 #include <string.h>
 #include <time.h>
-#endif  // defined(__cplusplus)
+#endif // defined(__cplusplus)
 
 #include <fcntl.h>
 #include <getopt.h>
 #include <linux/limits.h>
 #include <sys/types.h>
 #include <unistd.h>
 
-#define TIME_DIFF(Tstart, Tend)                                                                    \
-    ((double)(1000000000L * ((Tend).tv_sec - (Tstart).tv_sec) + (Tend).tv_nsec - (Tstart).tv_nsec) \
-     / 1000000000L)
+#define TIME_DIFF(Tstart, Tend)                                                \
+  ((double)(1000000000L * ((Tend).tv_sec - (Tstart).tv_sec) + (Tend).tv_nsec - \
+            (Tstart).tv_nsec) /                                                \
+   1000000000L)
 
 struct dyad_mod_ctx {
-    flux_msg_handler_t **handlers;
-    dyad_ctx_t *ctx;
+  flux_msg_handler_t **handlers;
+  dyad_ctx_t *ctx;
 };
 
 const struct dyad_mod_ctx dyad_mod_ctx_default = {NULL, NULL};
 
 typedef struct dyad_mod_ctx dyad_mod_ctx_t;
 
-static void dyad_mod_fini (void) __attribute__ ((destructor));
+static void dyad_mod_fini(void) __attribute__((destructor));
 
-void dyad_mod_fini (void)
-{
-    flux_t *h = flux_open (NULL, 0);
+void dyad_mod_fini(void) {
```

Comment:
Formatting is inconsistent.

Reply:
This will be handled by autoformatting (i.e., #124)